sdk/storage/Azure.Storage.Files.Shares/tests/FileTestBase.cs

```diff
@@ -249,7 +258,7 @@ public class DisposingShare : IAsyncDisposable
 
             public static async Task<DisposingShare> CreateAsync(ShareClient share, IDictionary<string, string> metadata)
             {
-                await share.CreateAsync(metadata: metadata, quotaInGB: 1);
+                await share.CreateAsync(metadata: metadata);
```

Comment:
Why do we want to remove this?

Reply:
Premium file share's minimum quota is much larger.